ChangeLog

```diff
@@ -1,5 +1,10 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS 2.x.x branch released 2017-xx-xx
+
+Changes
+   * Clarified the documentation of mbedtls_ssl_setup.
```

Comment:
Afaik, we usually write the ChangeLog in imperative style.

Reply:
Since the last commit was a changelog, I force-pushed an amendment of the last commit with “Clarify” instead of ”Clarified“.